hepstats/hypotests/calculators/asymptotic_calculator.py

```diff
@@ -246,8 +248,14 @@ def qalt(self, poinull, poialt, onesided, onesideddiscovery) -> np.ndarray:
         """
         nll_poinull_asy = self.asimov_nll(poinull, poialt)
         nll_poialt_asy = self.asimov_nll(poialt, poialt)
-        return self.q(nll1=nll_poinull_asy, nll2=nll_poialt_asy, poi1=poinull, poi2=poialt,
-                      onesided=onesided, onesideddiscovery=onesideddiscovery)
+        return self.q(
```

Comment:
Many of these changes must be formatting. Would you consider using `Black` for formatting given that many other Scikit-HEP packages are using it? Just for homogeneity.

Reply:
I actually applied the `Black` formatting.

Reply:
Oh, great. I did not check that :S. So add the badge as you see for example in https://github.com/scikit-hep/particle?

You can also use a pre-commit, see https://github.com/scikit-hep/particle/blob/master/.pre-commit-config.yaml ...

Reply:
What is the pre-commit doing actually?

Reply:
If you install it then black is run when you `git commit`, so you're always good to go before pushing to GitHub. It allows you to do the formatting "on the fly". It works like our GitLab formatting and copyright checks in LHCb, really.